registry/config_file_test.go

```diff
@@ -0,0 +1,161 @@
+package registry
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
+	"testing"
+
+	"github.com/docker/docker/pkg/homedir"
+)
+
+func pass() {
+	pc := make([]uintptr, 10)
+	runtime.Callers(0, pc)
```

Comment:
But doing this for all tests is simply add `-v` for `go test`. Yeah, I'm not in favor of some output in middle of test-suite for now. And personally I'm not sure why I want to see passed messages at all, but this is discussion for other PR.